tests/models/github_test.py

```diff
@@ -8,7 +8,7 @@
 
 from pythoncz import app
 from pythoncz.models import github as github_module
-from tests.models import github_test_fixtures as fixtures
+import github_test_fixtures as fixtures
```

Comment:
I just tried this and it works 😄 Ideally the tests would be probably simpler, maybe not needing the file at all, IDK